src/test/resources/com/microsoft/jenkins/azuread/integrations/casc/configuration-as-code.yml

```
@@ -0,0 +1,39 @@
+jenkins:
+  authorizationStrategy:
+    azureAdMatrix:
+      permissions:
+        - "Agent/Build:authenticated"
+        - "Agent/Configure:name (upn)"
+        - "Agent/Connect:name (upn)"
+        - "Agent/Create:name (upn)"
+        - "Agent/Delete:name (upn)"
+        - "Agent/Disconnect:name (upn)"
+        - "Credentials/Create:name (upn)"
+        - "Credentials/Delete:name (upn)"
+        - "Credentials/ManageDomains:name (upn)"
+        - "Credentials/Update:name (upn)"
+        - "Credentials/View:name (upn)"
+        - "Job/Build:authenticated"
+        - "Job/Cancel:name (upn)"
+        - "Job/Configure:name (upn)"
+        - "Job/Create:name (upn)"
+        - "Job/Delete:name (upn)"
+        - "Job/Discover:name (upn)"
```

this seems strange to me, why is upn added to all these permissions? shouldn't they be the same as the standard permissions?

---

If use this plugin to add an Azure user and give it some permissions, the `config.xml` file will add something like `<permission>hudson.model.Computer.Build:jie (jie@aaa.com)</permission>`

---

ah I see, misunderstood thought it was static, maybe put real UPNs in there and names?
```suggestion
        - "Job/Discover:bob (bob@contoso.com)"
```

---

done